docs: additional postinstall guides

secretmango opened the pull request:
reordered the GRUB password guide, to show the "dont worry" message earlier and the command directly after the "here is the command" message, improving clarity.

fixed incorrect polkit guide, following a recent Fedora Change request. Also removed incorrect "flatpak" group which does not do anything. This would be a group that could be added manually.

added a guide on how to add an SELinux confined user, linking to upstream docs and avoiding duplication. #338

added a guide on how to use QEMU/libvirt without root, via the unprivileged QEMU user session

secretmango committed:
docs: additional postinstall guides

fixed incorrect polkit guide, following a recent Fedora Change request.

added a guide on how to add an SELinux confined user

added a guide on how to use QEMU/libvirt without root, via the unprivileged QEMU user session

RoyalOughtness commented on POSTINSTALL-README.md:
```diff
@@ -48,7 +48,9 @@ Documentation is available [here](https://github.com/secureblue/secureblue/blob/
 
 Setting a GRUB password helps protect the device from physical tampering and mitigates various attack vectors, such as booting from malicious media devices and changing boot or kernel parameters.
 
-To set a GRUB password, use the following command. By default, the password will be required when modifying boot entries, but not when booting existing entries.
```
This seems fine how it was

secretmango replied:
if you think so. If I "resolve" this, will it be changed automatically?

RoyalOughtness replied:
no

RoyalOughtness commented on POSTINSTALL-README.md:
````diff
@@ -66,9 +68,9 @@ Creating a dedicated wheel user and removing wheel from your primary user helps
-1. ```adduser admin```
-2. ```usermod -aG wheel admin```
-3. ```passwd admin```
+1. ```sudo adduser admin```
````
sudo is implied, this isn't really necessary.

secretmango replied:
then the commands below are not correct. Also, the 4th command works without sudo.

I also prefer highly if given commands work. That way, it would be a barrier to users "to know that stuff". This includes knowing that `reboot` is `systemctl reboot` which is done via polkit and doesnt need sudo... Which I wouldnt think users need to know.

RoyalOughtness commented on POSTINSTALL-README.md:
```diff
@@ -117,3 +124,22 @@ The custom rule allows the group`diskadmin` to do the actions for unlocking and
+- `staff_u`
+- `user_u`
+
+## Use a virtual machine without root
```
this is not a postinstall step, maybe an FAQ item.

secretmango replied:
okay, will add an FAQ instead

RoyalOughtness commented on POSTINSTALL-README.md:
```diff
@@ -117,3 +124,22 @@ The custom rule allows the group`diskadmin` to do the actions for unlocking and
+
+At the current state, unlike on Android, a user account and all it's processes are not protected by SELinux on Fedora.
+
+As this may cause breakages, [follow these guides](https://fedoraproject.org/wiki/SELinux/ConfinedUsers#Assign_a_SELinux_user_to_an_existing_Linux_user) on how to add a new user account and confine it.
```
We need to incorporate these steps into the guide for user provisioning instead of just linking to the fedora docs. it should be one seamless set of steps

secretmango replied:
okay, I wanted to avoid duplication and make it more maintainable that way. The link is also directly to the steps needed, the other step (creating the user) is included there.

So this should not be in the FAQ but stay here?

secretmango committed:
fixes

added the full SELinux confined users setup, only linking to upstream Fedora docs for information, not instructions.

Added a dedicated chapter for user setup, as it sounded like this was the idea

secretmango commented:
I added some updates!

RoyalOughtness commented on POSTINSTALL-README.md:
```diff
@@ -1,4 +1,4 @@
-# secureblue
+# Secureblue Post Install Recommendations
```
this was fine how it was

RoyalOughtness commented on POSTINSTALL-README.md:
````diff
@@ -78,35 +81,33 @@ Creating a dedicated wheel user and removing wheel from your primary user helps
 ```
-cat >> /etc/polkit-1/rules.d/80-udisks2.rules <<EOF
-polkit.addRule(function(action, subject) {
-    if (action.id == "org.freedesktop.udisks2.encrypted-unlock-system" || action.id == "org.freedesktop.udisks2.filesystem-mount-system" &&
````
why was this removed?

RoyalOughtness replied:
the comment I made was just about VMs :)

RoyalOughtness commented:
https://github.com/secureblue/secureblue/issues/338#issuecomment-2259311313
